sources/MSBulletin_V2.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Source file for Microsoft Bulletin information
+#
+# Software is free software released under the "Modified BSD license"
+#
+# Copyright (c) 2016    Pieter-Jan Moreels - pieterjan.moreels@gmail.com
+
+# Imports
+import copy
+import datetime
+import time
+import json
+import requests
+
+from collections import defaultdict
+
+from lib.Config  import Configuration as conf
+from lib.Source  import Source
+
+
+SOURCE_NAME = "msbulletin"
+SOURCE_FILE = "https://portal.msrc.microsoft.com/api/security-guidance/en-us/"
+
+
+# Get Ms Bulletins in Json Format
+def get_msbulletin(url, from_date= "01/01/1900" , to_Date= None ):
+    headers={"Accept": "application/json, text/plain, */*",
+    "Content-Type": "application/json;charset=utf-8",
+    "Referer": "https://portal.msrc.microsoft.com/en-us/security-guidance"
+    }
+    query = {
+          'familyIds': [], 'productIds': [], 'severityIds': [], 'impactIds': [],
+          'pageNumber': 1, 'pageSize': 50000,
+          'includeCveNumber':True, 'includeSeverity': True, 'includeImpact': True,
+          'orderBy': 'publishedDate', 'orderByMonthly': 'releaseDate',
+          'isDescending': True, 'isDescendingMonthly': True,
+          'queryText': '', 'isSearch': False, 'filterText': '',
+          'fromPublishedDate': from_date
+    }
+
+    if to_Date:
+      query['toPublishedDate'] = to_date
+      
+    post = requests.post(url,  headers=headers, data=json.dumps(query) )
+    if post:
+      return post.json()
+    else:
```

Comment:
Actually you could write this this way rather than with `if post`:
`return post and post.json() or {}` 
This way you never return None but a dict eventually empty